drivers/i2c/i2c_sbcon.c

```diff
@@ -116,8 +116,9 @@ static const struct i2c_sbcon_config i2c_sbcon_dev_cfg_##_num = {	\
 	.sbcon		= (void *)DT_INST_REG_ADDR(_num), \
 };									\
 									\
-DEVICE_AND_API_INIT(i2c_sbcon_##_num, DT_INST_LABEL(_num), \
+DEVICE_DT_INST_DEFINE(num,						\
 	    i2c_sbcon_init,						\
+	    NULL,							\
```

Comment:
The documented parameter when power management is not supported is `device_pm_control_nop`.  Let's please use that until we decide is unnecessary and make everything consistent with intent.

Reply:
Ok, will update the other PRs to match.